freqtrade/rpc/telegram.py

```diff
@@ -129,7 +129,7 @@ def _init_keyboard(self) -> None:
             r'/weekly$', r'/weekly \d+$', r'/monthly$', r'/monthly \d+$',
             r'/forcebuy$', r'/forcelong$', r'/forceshort$',
             r'/forcesell$', r'/forceexit$',
-            r'/edge$', r'/health$', r'/help$', r'/version$'
+            r'/edge$', r'/health$', r'/help$', r'/version$', r'/marketdir$'
```

Comment:
i think this is pointless.

it'll allow you to create a button `/marketdir` (without anything after it) - but if you call `/marketdir` without argument, it'll do nothing based on the handler, which requires 1 argument.